adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx

```diff
@@ -173,16 +229,16 @@ export default class AnimalCreateForm
             } catch (error) {
                 this.showErrorAlert(error);
             } finally {
-                this.setState({ isSubmitting: false });
+                this.setState({isSubmitting: false});
             }
         } else {
-            this.setState({ showInvalidFormAlert: true });
+            this.setState({showInvalidFormAlert: true});
         }
         event.preventDefault();
     }
 
     private isFormValid() {
-        const { animal } = this.state;
+        const {animal} = this.state;
```

Comment:
I was trying to use a consistent spacing rule here for objects https://eslint.org/docs/rules/object-curly-spacing. Leaving a space after/before brackets.

Didn't have time to enforce it in the projects coding style. It's specified in the `.eslintrc` file, and can be checked with `npm run lint`.

I'm happy to use either one way or the other, but we should be consistent.

What are your thoughts?